lib/nolibc/Makefile.uk

```diff
@@ -18,6 +18,9 @@ LIBNOLIBC_NO_BUILTINS		+= -fno-builtin-vsscanf
 CFLAGS-$(CONFIG_LIBNOLIBC)		+= $(LIBNOLIBC_NO_BUILTINS)
 CXXFLAGS-$(CONFIG_LIBNOLIBC)		+= $(LIBNOLIBC_NO_BUILTINS)
 
+CFLAGS-$(CONFIG_LIBNOLIBC)		+= -ffreestanding
```

Comment:
I think it should be enough to add it to the `C{,XX}FLAGS` of nolibc (and musl in the other repository).

Reply:
@mschlumpp, it is added to the `CFLAGS` / `CXXFLAGS` of nolibc (using `CONFIG_LIBNOLIBC`). What is the issue?

Reply:
Agree, makes sense to have the flag for nolibc only, but also require it in musl (and any other c lib).

Reply:
@razvand That would be the case with:
`LIBNOLIBC_CFLAGS-y`

Reply:
Gotcha, I fixed it. @marcrittinghaus, @mschlumpp